apple/Sources/FerrostarSwiftUI/Views/Controls/FerrostarButton.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+public struct FerrostarButtonStyle: ButtonStyle {
```

Comment:
I think`StyledActionButton` might be a more descriptive type name here. There isn't really anything Ferrostar-specific; it's just got our `FerrostarButtonStyle` applied, a label, etc.

Reply:
I'm fine with `StyledActionButton`, but my intention was to prepend Ferrostar so it's easy to follow if someone want's to use the same button style for their own navigation actions(s) outside of the ferrostar scope. Similar to the ViewModifiers, prepending with Ferrostar makes it easy to understand in a final implementation.

Reply:
How about `NavigationUIButton`